### DISCUSSION TITLE
Welcome to nts Discussions!

### DISCUSSION THREAD

**Discussion by tweeeeeak** [tweeeeeak/nts#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by julianobeccari** [comment]
Hi
I 've just bought the NTS-1 mkii and I discover that all thes beautiful plugin work only in mk1 so I wanted to know when du you plan to make version for it! 

Best wishes and prog lines to you!

Can I learn a little bit of progtramming? I'm just a musician... 

Thanks

Juliano

**Comment by frbox** [comment]
Hi! Do you plan to adapt your "RoAT" to the NTS1 mk2? Would that be a complex project? Keep up the great work!
